content/en/docs/release-notes/release-notes-0.14.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Release Notes"
+linkTitle: "v0.14"
+weight: 140
+type: "docs"
+---
+
+In `v0.14` we've focused on a number of areas:
+
+## Webhook changes
+
+The webhook component has now been made required, meaning any users that previously used the `no-webhook` variant will
+need to upgrade their installations to enable the webhook. This will be done automatically upon upgrading to the `v0.14`
+release, so no additional action is required.
+
+If you have issues running the webhook in your environment, we'd like to hear from you! We are aware of issues relating
+to firewall rules from the Kubernetes API server to the webhook pod(s) - we would like to gather together a corpus of
+configuration snippets that can be used to ensure the webhook is successfully deployed in these environments too.
```

Comment:
```suggestion
If you encounter problems running the webhook in your environment, we'd like to hear from you! 
We are aware of the following issues:
 * [firewall rules from the Kubernetes API server to the webhook pod(s)](INSERT_LINK)
 
If you have difficulty deploying the webhook, please give feedback and information about your environment.
We will try our best to offer work arounds and fixes in future releases. 
```

Reply:
These issues aren't 'known issues' as in, bugs in cert-manager, rather they are additional steps required in different deployment environments (e.g. users running 'private' GKE clusters need to explicitly whitelist some traffic). I wouldn't call these 'fixes' or 'things we are aware of' in the sense of something we need to fix - rather it is something we need to document and provide guidance on, hence my original wording around building a corpus. Not sure what you think in light of that :)